R/wrappers.R

```diff
@@ -215,10 +231,20 @@ t_stat <- function(x, formula,
 #' @param ... Additional arguments for [chisq.test()][stats::chisq.test()].
 #'
 #' @examples
-#' # chisq test for comparing number of cylinders against automatic/manual
-#' mtcars %>%
-#'   dplyr::mutate(cyl = factor(cyl), am = factor(am)) %>%
-#'   chisq_test(cyl ~ am)
+#' # chi-squared test of independence for college completion 
+#' # status depending on one's self-identified income class
+#' chisq_test(gss, college ~ finrela)
+#' 
+#' # chi-squared goodness of fit test on whether self-identified 
+#' # income class follows a uniform distribution
+#' chisq_test(gss, 
+#'            response = finrela,
+#'            p = c("far below average" = .167,
```

Comment:
@simonpcouch Could we switch all of these to be `1/6` instead? Doesn't seem right to say it's uniform and then have one level be different at .165.